WordPress/src/main/java/org/wordpress/android/widgets/AppReviewManager.kt

```diff
@@ -94,6 +127,42 @@ object AppRatingDialog {
         }
     }
 
+    /**
+     * Called when a post is published. We use this to determine which users will see the in-app review prompt.
+     */
+    fun onPostPublished() {
+        if (shouldShowInAppReviewsPrompt()) return
+        if (AppPrefs.getPublishedPostCount() < TARGET_COUNT_POST_PUBLISHED) {
+            AppPrefs.incrementPublishedPostCount()
+            AppLog.d(T.UTILS, "In-app reviews counter for published posts: ${AppPrefs.getPublishedPostCount()}")
+        }
+    }
+
+    /**
+     * Called when a notification is received. We use this to determine which users will see the in-app review prompt.
+     */
+    fun onNotificationReceived(note: Note) {
+        if (shouldShowInAppReviewsPrompt()) return
+        val shouldTrack = note.isUnread && (note.isLikeType || note.isCommentType || note.isFollowType)
+        if (shouldTrack && AppPrefs.getInAppReviewsNotificationCount() < TARGET_COUNT_NOTIFICATIONS) {
+            AppPrefs.incrementInAppReviewsNotificationCount()
+            AppLog.d(T.UTILS, "In-app reviews counter for notification: ${AppPrefs.getInAppReviewsNotificationCount()}")
+        }
+    }
+
+    /**
+     * Check whether the in-app reviews prompt should be shown or not.it after its last sh
```

Comment:
Small nit: typo at the end of the sentence here

Reply:
Good catch! Fixed: https://github.com/wordpress-mobile/WordPress-Android/pull/20891/commits/e4689656c17e6da20cd20b908155f4df1250e524